### PR TITLE
Be nc news 9

### PR DESCRIPTION
**Pull request by yaahcd**

### CODE REVIEW

File: __tests__/app.test.js
Comment (Slyose) [inline __tests__/app.test.js:1]: You shouldn't have code for two different endpoints on one PR - there's new tests for deleting and patching comments here. Make sure to keep it at one ticket per PR!

File: controller/controller.js [hunk 1]
```diff
@@ -29,3 +33,32 @@ exports.getArticlesById = (req, res, next) => {
...
+  const id = req.params.comment_id 
+
+  const promises = [checkCommentIdExists(id), deleteSelectedComment(id)]
+
+  Promise.all(promises).then((completedPromises) => {
+    const emptyObj = completedPromises
+    res.status(204).send(emptyObj)
+  })
+  .catch(next)
+}
```
Comment (Slyose) [inline controller/controller.js]: You don't need to send anything back on a 204 - just the 204 status is fine, as there's no data to send back.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -104,3 +104,74 @@ test('200: Should return objects without body property', () => {
...
+			.then(({ body }) => {
+				expect(body.msg).toBe('Bad request');
+			});
+	});
+});
+describe('DELETE /api/comments/:comment_id', () => {
+	test('204: Should return empty object if passed valid comment id', () => {
+		return request(app).delete('/api/comments/2').expect(204).then(({body}) => {
+			expect(body).toEqual({})
+		})
```
Comment (Slyose) [inline __tests__/app.test.js]: No need to test for the empty body here - just a status 204 is fine!

**Review by Slyose — COMMENTED** [review]
Very well done. A couple of comments to address, but feel free to merge solo afterwards.

Btw, this ticket was confusing to review as it had code for two different tickets. Make sure to avoid doing this, and keep your PRs to one ticket each.
